src/Jupyter/Visualization/StateDisplayEncoders.cs

```diff
@@ -268,17 +303,65 @@ public StateVectorToHtmlResultEncoder(IConfigurationSource configurationSource)
                             _ => throw new ArgumentException($"Unsupported style {ConfigurationSource.PhaseDisplayStyle}")
                         };
 
-                        return FormattableString.Invariant($@"
-                            <tr>
-                                <td>$\left|{basisLabel}\right\rangle$</td>
-                                <td>${amplitude.Real:F4} {(amplitude.Imaginary >= 0 ? "+" : "")} {amplitude.Imaginary:F4} i$</td>
+                        count = count + 1;
+
+                        //different options for displaying measurement style
+                        var measurementHistogram = ConfigurationSource.MeasurementDisplayHistogram;
+
+                        var measurementPrecision = ConfigurationSource.MeasurementDisplayPrecision;
+    
+                        var measurementCell = ConfigurationSource.MeasurementDisplayStyle switch
+                        {
+                            MeasurementDisplayStyle.None => String.Empty,
+                            //measurement_off => 1;
+                            MeasurementDisplayStyle.BarOnly => FormattableString.Invariant($@"
                                 <td>
                                     <progress
                                         max=""100""
                                         value=""{System.Math.Pow(amplitude.Magnitude, 2.0) * 100}""
                                         style=""width: 100%;""
                                     >
                                 </td>
+                            "),
+                            MeasurementDisplayStyle.BarAndNumber => FormattableString.Invariant($@"
+                                <td>
+                                    <progress
+                                        max=""100""
+                                        value=""{System.Math.Pow(amplitude.Magnitude, 2.0) * 100}""
+                                        style=""width: 100%;""
+                                    > 
+                                    <td>
+                                    <p id=""round${count}""> 
+                                    <script>
+                                    var num = {System.Math.Pow(amplitude.Magnitude, 2.0) * 100};
+                                    num = num.toFixed({measurementPrecision});
+                                    var num_string = num + ""%"";
+                                     document.getElementById(""round${count}"").innerHTML = num_string;
+                                    </script> </p>
```

Comment:
I'd suggest that it would be simpler here to generate the measurement percentages in C# rather than injecting JavaScript here, since you don't have any hook for those measurement probabilities to change on the client side.
That would also allow you to remove the dependency on `count` entirely, actually. 
```suggestion
                                    <p id=""round${count}"">
                                        {(
                                            System.Math.Pow(amplitude.Magnitude, 2.0) * 100
                                        ).ToString($"F{measurementPrecision}")}
                                    </p>
```